tests/test-cases.js

```diff
@@ -1897,6 +1897,114 @@ var CUSTOM_TESTS = [
 			});
 		}
 	},
+	{
+		description: "Pause and resume works for chunks with NetworkStreamer (for Bug #736)",
```

Comment:
I do not think is needed to include the bug # in the test. It can be directly obtained from git history